cs251tk/webapp/web_cli.py

```diff
@@ -0,0 +1,129 @@
+import os
+
+from ..common import chdir
+from ..student import stash, pull, checkout_date
+from ..student.markdownify.process_file import process_file
+from PyInquirer import style_from_dict, Token, prompt
+
+
+def check_student(student, spec, basedir):
+    files = []
+    if os.path.exists('{}/{}'.format(student, spec['assignment'])):
+        with (chdir('{}/{}'.format(student, spec['assignment']))):
+            for file in spec['files']:
+
+                result = process_file(file['filename'],
+                                      steps=file['commands'],
+                                      options=file['options'],
+                                      spec=spec,
+                                      cwd=os.getcwd(),
+                                      supporting_dir=os.path.join(basedir, 'data', 'supporting'),
+                                      interact=False,
+                                      basedir=basedir,
+                                      spec_id=spec['assignment'],
+                                      skip_web_compile=False)
+
+                if 'web' in file['options']:
+                    if result['missing']:
```

Comment:
```python
description = file['filename']

if result['missing']:
    if 'optional' in file['options']:
        description = '{} MISSING (OPTIONAL)'.format(file['filename'])
    else:
        description = '{} MISSING'.format(file['filename'])

files = files + [description]
```

(to replace the inside of this whole `if web in file[options]` block)

Reply:
@maxnz you opted against this suggestion?

Reply:
I must have included it and then deleted it before committing.